connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java

```diff
@@ -257,13 +257,23 @@ public WorkerCoordinatorMetrics(Metrics metrics, String metricGrpPrefix) {
 
             Measurable numConnectors = new Measurable() {
                 public double measure(MetricConfig config, long now) {
```

Comment:
@SupermanScott This change looks ok, but this is failing checkstyle:

```
:connect:runtime:checkstyleMain
[ant:checkstyle] /Users/ewencp/kafka.git/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java:280:1: File contains tab characters (this is the first instance).
[ant:checkstyle] /Users/ewencp/kafka.git/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java:282:21: '}' should be on the same line.
[ant:checkstyle] /Users/ewencp/kafka.git/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java:293:21: '}' should be on the same line.
:connect:runtime:checkstyleMain FAILED
```

Looks like simple indentation cleanup. You can easily check that these are fixed by running `./gradlew :connect:runtime:checkstyleMain`.